buildpack.toml

```diff
@@ -16,5 +16,8 @@ api = "0.7"
 [[stacks]]
   id = "io.buildpacks.stacks.bionic"
 
+[[stacks]]
+  id = "io.buildpacks.stacks.jammy"
+
```

Comment:
Could we switch to using a wildcard stack id? Here's an example:

https://github.com/paketo-buildpacks/bundle-install/blob/d210db4fc698860eb58cf59fc080d6a403d5a1c7/buildpack.toml#L19-L20

Reply:
I think a wildcard should work. After all, what it really means is that `pip-install` should work wherever its dependencies are met.

Reply:
done